src/bioregistry/external/togoid/processed.json

```diff
@@ -195,6 +198,7 @@
   },
   "Chebi": {
     "catalog": "nbdc00027",
+    "description": "ChEBI is a collection of data on entities of biological interest.\r\nThis database concentrates on assembling data on smaller chemical compounds, such as atoms, molecules, ions, radicals etc. These should be produced in nature or be synthetic molecules affecting biological processes in living organisms.\r\nChEBI does not generally include molecules directly encoded by the genome.\r\nThe database contains 31,651 entries with annotations.\r\nA keyword search option is available.\r\nIn addition to this an advanced search option, which includes a structure drawing tool, is provided.\r\nThe ChEBI database can be searched individually or in combination with other databases.",
```

Comment:
Not clear if keeping in newlines here is good or can be problematic, but at least we should normalize the `\r` carriage returns out of it.

Reply:
otherwise, all good! thanks!

Reply:
Okay, I removed all of those, so we just get a space instead of all new lines / carriage returns which I think should be fine.